lib/src/tasks/test/cli.dart

```diff
@@ -38,6 +38,9 @@ class TestCli extends TaskCli {
     ..addFlag('functional',
         defaultsTo: defaultFunctional,
         help: 'Includes the functional test suite.')
+    ..addFlag('disable-serve-std-out',
+        defaultsTo: defaultDisableServeStdOut,
```

Comment:
Instead of using `defaultDisableServeStdOut`, we could use `config.disableServeStdOut` so that if a project configures it to a different value, the default CLI behavior will match